shared/menubar/index.desktop.js

```diff
@@ -80,95 +54,93 @@ class MenubarRender extends Component<Props, State> {
+  _menuView(title: string, iconType: Kb.IconType, count: number) {
     return (
-      <Box2 direction="horizontal" style={{width: '100%'}}>
-        <Box style={{marginRight: globalMargins.xsmall, position: 'relative'}}>
-          <Icon type={iconType} color={globalColors.black_20} fontSize={20} />
-          {!!count && <Badge badgeNumber={count} badgeStyle={{position: 'absolute', left: 14, top: -2}} />}
-        </Box>
-        <Text className="title" type="Body" style={collapseStyles([{color: undefined}])}>
+      <Kb.Box2 direction="horizontal" style={{width: '100%'}}>
+        <Kb.Box style={{marginRight: Styles.globalMargins.xsmall, position: 'relative'}}>
+          <Kb.Icon type={iconType} color={Styles.globalColors.black_20} fontSize={20} />
+          {!!count && <Kb.Badge badgeNumber={count} badgeStyle={{position: 'absolute', left: 14, top: -2}} />}
+        </Kb.Box>
+        <Kb.Text className="title" type="Body" style={Styles.collapseStyles([{color: undefined}])}>
           {title}
-        </Text>
-      </Box2>
+        </Kb.Text>
+      </Kb.Box2>
     )
   }
 
   _menuItems(countMap: Object) {
     return [
-      ...(flags.walletsEnabled
+      ...(Flags.walletsEnabled
         ? [
             {
               title: 'Wallet',
-              view: this._menuView('Wallet', 'iconfont-nav-wallets', countMap[walletsTab] || 0),
-              onClick: () => this.props.openApp(walletsTab),
+              view: this._menuView('Wallet', 'iconfont-nav-wallets', countMap[Tabs.walletsTab] || 0),
+              onClick: () => this.props.openApp(Tabs.walletsTab),
             },
           ]
         : []),
       {
         title: 'Git',
-        view: this._menuView('Git', 'iconfont-nav-git', countMap[gitTab] || 0),
-        onClick: () => this.props.openApp(gitTab),
+        view: this._menuView('Git', 'iconfont-nav-git', countMap[Tabs.gitTab] || 0),
+        onClick: () => this.props.openApp(Tabs.gitTab),
       },
       {
         title: 'Devices',
-        view: this._menuView('Devices', 'iconfont-nav-devices', countMap[devicesTab] || 0),
-        onClick: () => this.props.openApp(devicesTab),
+        view: this._menuView('Devices', 'iconfont-nav-devices', countMap[Tabs.devicesTab] || 0),
+        onClick: () => this.props.openApp(Tabs.devicesTab),
       },
       {
         title: 'Settings',
-        view: this._menuView('Settings', 'iconfont-nav-settings', countMap[settingsTab] || 0),
-        onClick: () => this.props.openApp(settingsTab),
+        view: this._menuView('Settings', 'iconfont-nav-settings', countMap[Tabs.settingsTab] || 0),
+        onClick: () => this.props.openApp(Tabs.settingsTab),
       },
       'Divider',
       ...(this.props.loggedIn ? [{title: 'Open main app', onClick: () => this.props.openApp()}] : []),
-      {title: 'Open folders', onClick: () => this.props.openApp(fsTab)},
+      {title: 'Open folders', onClick: () => this.props.openApp(Tabs.fsTab)},
```

Comment:
Should this be "Open Files"?

Reply:
The design says "Open folders", but I think you're right, it should be "files".